pkg/cli/workspaces/connection.go

```diff
@@ -92,14 +94,26 @@ func (ws Workspace) ConnectionConfig() (ConnectionConfig, error) {
 	}
 }
 
-// Connect attempts to create a connection to the workspace using the connection configuration and returns the
+// Connect attempts to create and test a connection to the workspace using the connection configuration and returns the
 // connection and an error if one occurs.
-func (ws Workspace) Connect() (sdk.Connection, error) {
+func (ws Workspace) Connect(ctx context.Context) (sdk.Connection, error) {
```

Comment:
👍

Reply:
yes, all current calls were followed by another call to test connection